### PR TITLE
Disable auto update on reenable/install

### PR DESCRIPTION
**Pull request by Eeems**
- Disable `update-engine.service` on install/reenable
- Decouple `sync.service` and `xochitl.service` as this means that either sync doesn't work, or xochitl starts when it is disabled.
  - Accomplished by adding a `manual-sync.service` and masking `sync.service`
  - `/opt/bin/xochitl` will also run `sync --service` in the background manually if running from a chroot and there is no sync service running
- Add `kill-xochitl` command as oxide stop events do not allow arguments

### CODE REVIEW

**Comment by Eeems** [comment]
Untested

File: package/kernelctl/package [hunk 1]
```diff
@@ -5,7 +5,7 @@
 pkgnames=(kernelctl)
 pkgdesc="Manage aftermarket kernels"
 url=https://toltec-dev.org/
-pkgver=0.1-2
+pkgver=0.1-4
```
Comment (Eeems) [inline package/kernelctl/package:8]: There is another PR with `-3` that will likely get merged before this one is.

**Comment by Eeems** [comment]
I can't test the `toltec-base` install because it's failing to fetch `open-remarkable-shutdown`. Even when I try to install that from a ipk file, it still fails. Not sure what is happening.

**Comment by Eeems** [comment]
Okay, I've reinstalled toltec and this seems to have resolved it. This seems to work for me now, but someone will need to test/confirm on a rM2.

**Review by LinusCDE — DISMISSED** [review]
Works fine

**Comment by Eeems** [comment]
I guess sync should be unmasked on uninstall?

**Comment by LinusCDE** [comment]
Wouldn't hurt I guess. Best to restore the original state on uninstall.

**Review by LinusCDE — APPROVED** [review]
Works fine. :+1:

**Comment by matteodelabre** [comment]
@Eeems Did this resolve #553?

**Comment by Eeems** [comment]
Most likely

**Pull request merged**